.github/workflows/build_rpms.yaml

```diff
@@ -0,0 +1,55 @@
+name: Build RPMs
+
+on:
+  # push:
+  schedule:
+  - cron: '0 0 * * *'
+
+jobs:
+  build_rpm_build_container:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        repository: ManageIQ/manageiq-rpm_build
+    - name: Set up registry credentials
+      if: ${{ github.event_name != 'pull_request' }}
+      run: |
+        echo "REGISTRY_USERNAME=${{ secrets.REGISTRY_USERNAME }}" >> $GITHUB_ENV
+        echo "REGISTRY_PASSWORD=${{ secrets.REGISTRY_PASSWORD }}" >> $GITHUB_ENV
+    - name: Run container build script
+      run: bin/build_container_image
+  build_rpms:
+    needs: build_rpm_build_container
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up registry credentials
+      run: |
+        echo "REGISTRY_USERNAME=${{ secrets.REGISTRY_USERNAME }}" >> $GITHUB_ENV
+        echo "REGISTRY_PASSWORD=${{ secrets.REGISTRY_PASSWORD }}" >> $GITHUB_ENV
+    - name: Build and push RPMs
+      run: |
+        echo $REGISTRY_PASSWORD | docker login docker.io --password-stdin --username $REGISTRY_USERNAME
+        mkdir options
+        echo "---" > options/options.yml
+        echo "rpm_repository:" >> options/options.yml
+        echo "  digitalocean_access_token: ${{ secrets.RPM_BUILD_DO_ACCESS_TOKEN }}" >> options/options.yml
+        echo "  s3_api:" >> options/options.yml
+        echo "    access_key: ${{ secrets.RPM_BUILD_S3_ACCESS_KEY " >> options/options.yml
+        echo "    secret_key: ${{ secrets.RPM_BUILD_S3_SECRET_KEY " >> options/options.yml
+        docker run docker.io/manageiq/rpm_build:latest build -v options:/root/OPTIONS --build-type nightly
+  notify_builders:
+    needs: build_rpms
+    runs-on: ubuntu-latest
+    steps:
+    - name: Notify manageiq-pods on RPM build
+      uses: peter-evans/repository-dispatch@v1
+      with:
+        token: ${{ secrets.DEPLOY_TOKEN }}
+        repository: ManageIQ/manageiq-pods
+        event-type: build
+        client-payload: |
+          { "repository": "${{ github.repository }}",
+            "pr_number": "${{ steps.regex-commit-message.outputs.group1 }}",
```

Comment:
This doesn't exist here - I assume you copied it from the website build, but you're missing the steps where it looks at the HEAD commit message to pull out the PR number: https://github.com/ManageIQ/manageiq.org/blob/master/.github/workflows/ci.yaml#L26-L32

Reply:
Yes, it was copied.  I didn't know what we want to put in here yet.